stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,98 @@
+    def Barrier(self, parties: int, action: Optional[Callable[..., Any]] = ..., timeout: Optional[int] = ...) -> Any: ...
+    # TODO: change return to Queue once a stub exists in multiprocessing.queues
+    def Queue(self, maxsize: int = ...) -> Any: ...
+    # TODO: change return to Queue once a stub exists in multiprocessing.queues
+    def JoinableQueue(self, maxsize: int = ...) -> Any: ...
+    # TODO: change return to SimpleQueue once a stub exists in multiprocessing.queues
+    def SimpleQueue(self) -> Any: ...
+    # TODO: change return to Pool once a stub exists in multiprocessing.pool
+    def Pool(
+        self,
+        processes: Optional[int] = ...,
+        initializer: Optional[Callable[..., Any]] = ...,
+        initargs: Tuple = ...,
+        maxtasksperchild: Optional[int] = ...
+    ) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to RawValue once a stub exists in multiprocessing.sharedctypes
+    def RawValue(self, typecode_or_type: Any, *args: Any) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to RawArray once a stub exists in multiprocessing.sharedctypes
+    def RawArray(self, typecode_or_type: Any, size_or_initializer: Union[int, Sequence[Any]]) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to Value once a stub exists in multiprocessing.sharedctypes
+    def Value(
+        self,
+        typecode_or_type: Any,
+        *args: Any,
+        lock: bool = ...
+    ) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to Array once a stub exists in multiprocessing.sharedctypes
+    def Array(
+        self,
+        typecode_or_type: Any,
+        size_or_initializer: Union[int, Sequence[Any]],
+        *,
+        lock: bool = ...
+    ) -> Any: ...
+    def freeze_support(self) -> None: ...
+    def get_logger(self) -> Logger: ...
+    def log_to_stderr(self, level: Optional[str] = ...) -> Logger: ...
+    def allow_connection_pickling(self) -> None: ...
+    def set_executable(self, executable: str) -> None: ...
+    def set_forkserver_preload(self, module_names: List[str]) -> None: ...
+    def get_context(self, method: Optional[str] = ...) -> BaseContext: ...
+    def get_start_method(self, allow_none: bool = ...) -> str: ...
+    def set_start_method(self, method: Optional[str] = ...) -> None: ...
+    @property
+    def reducer(self) -> str: ...
+    @reducer.setter
+    def reducer(self, reduction: str) -> None: ...
+    def _check_available(self) -> None: ...
```

Comment:
There are also a bunch of concrete contexts (e.g. SpawnContext), but I'm not sure we need those; they are not documented.